main/messages/new_message.php

```diff
@@ -149,6 +149,7 @@ function manageForm($default, $select_from_user_list = null, $sent_to = '', $tpl
                     [],
                     [
                         'multiple' => 'multiple',
+                        'delay' => '1000',
```

Comment:
This should be probably an int instead of a text.

Reply:
int casting missing

$delay = (int) $this->getAttribute('delay'); 

$delay will be 0 when return null, so change this validation

$delay = !empty($delay) ? "delay: $delay, " : '';

to

$delay = (0 !== $delay) ? "delay: $delay, " : '';